recipes/perl-tie-refhash/1.39/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "perl-tie-refhash" %}
+{% set version = "1.39" %}
+{% set sha256 = "b0b80ef571e7dadb726b8214f7352a932a8fa82af29072895aa1aadc89f48bec" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: Tie-RefHash-1.39.tar.gz
+  url: https://cpan.metacpan.org/authors/id/F/FL/FLORA/Tie-RefHash-1.39.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl >5.22,<6.0
+    - perl-extutils-makemaker
+
+  run:
+    - perl >5.22,<6.0
+
+test:
+  imports:
+    - Tie::RefHash
+    #- Tie::RefHash::Nestable
+
+about:
+  home: http://metacpan.org/pod/Tie::RefHash
+  license: unknown
```

Comment:
Can you figure out the License, otherwise we are not allowed to redistribute the package :(

Reply:
updated to perl5 license. This is a core module and hence can be redistributed and/or modified under the same terms as Perl5 itself
;)